### DISCUSSION TITLE
Github Projects Beta: Option to set a default value when adding an issue.

### DISCUSSION THREAD

**Discussion by mikelyncheski** [community/community#8076] [Projects and Issues] [answered]
I just watched the youtube video by @azenMatt (https://www.youtube.com/watch?v=SI1ra-XHWHM) and we are buying into the idea of switching to Github Project Beta for our boards.

However, when setting up our first significant project, it was time-consuming to have to set default values on certain fields.  The best example was setting the status to "backlog".

This is something you should be able to do in a day Matt! :wink:

**Comment by azenMatt** [comment, marked as answer]
Hi @mikelyncheski - I'm glad you enjoyed the video! 

We do have a built in workflow (menu on the top right) that allows you to assign a status when an item is added, does this help at all?

![image](https://user-images.githubusercontent.com/7584089/143446246-ae5961f6-e231-4efb-a99d-d8d5b58beff6.png)


We will have more options here soon to go past Status.

**Reply by sd-sharp** [reply]
Hello.
This comment was on 2021.

Do we have more options to set other columns defaults ...and not just Status ?

**Reply by grossmj** [reply]
Indeed, setting another column, for instance priority, would be great!

**Reply by danielgomezrico** [reply]
Hi @azenMatt

Its been 2 years and you mentioned that this should exist soon… is there any reason for not adding it? 🤓

**Reply by raphaelsamyfacily** [reply]
sadly for something that has the money that MS has its moving way too slow, they could easily make many people change from jira etc.. to github projects if just added "simple" features

**Reply by giorgosmic** [reply]
Would be great if we could add a value to custom field from an API

**Reply by BlueSkunka** [reply]
This feature is to limited and should allow to set default value for any columns to improve project workflow !

**Reply by rfc2822** [reply]
Being able to set a default value for custom fields would really help a lot.

**Comment by mikelyncheski** [comment]
Yes, this is perfect!!

Thanks

**Comment by nastanford** [comment]
I want to be able to say new issues or items I create get assigned to me be default.   However, when someone else adds issues or items then don't assign it to me. Hope taht is coming soon.

**Comment by danielgomezrico** [comment]
I dont see the selected answer as a valid response for the questions, this does not respond the issue.

How can we do it?

**Comment by 1112779648** [comment]
I did not ask you for anything, and if you wished, I could have asked for it last night, but the one who wanted was one of my clients and a trustworthy person
